### ISSUE TITLE
this project is shown as deprecated on npm registry

### ISSUE THREAD

**Issue by rodgomes** [container-registry/backstage-plugin-harbor#240]
I noticed that this package is now showing as deprecated on the npm registry, which might bring security concerns for some people that are planning to use it. 

is that the case, or was it because the handler changed from bestseller to container-registry? 
https://www.npmjs.com/package/@bestsellerit/backstage-plugin-harbor

<img width="1459" alt="Screenshot 2023-08-09 at 13 14 50" src="https://github.com/container-registry/backstage-plugin-harbor/assets/131717/17852023-94a9-49b6-89df-2b538d40739a">

**Comment by Vad1mo** [comment]
yes, exactly. However I didn't yet made it possible to publish the packge to NPM.js set

**Comment by johanhammar** [comment]
Can we expect a new release soon? PRs like https://github.com/container-registry/backstage-plugin-harbor/pull/242 would probably help many of us :)

**Comment by Vad1mo** [comment]
need some help on that one https://github.com/container-registry/backstage-plugin-harbor/issues/263

**Comment by psmietanaa** [comment]
Do you have a timeline when this will be implemented @Vad1mo?
